internal/action/main.go

```diff
@@ -7,49 +7,74 @@ import (
 	"io/ioutil"
 	"os"
 	"path"
-	"strings"
 
 	"github.com/hashicorp/go-tfe"
 	"github.com/hashicorp/terraform-exec/tfexec"
 	"github.com/sethvargo/go-githubactions"
-	"github.com/takescoop/terraform-cloud-workspace-action/internal/action/inputs"
 	"github.com/takescoop/terraform-cloud-workspace-action/internal/tfconfig"
 	"github.com/takescoop/terraform-cloud-workspace-action/internal/tfeprovider"
 	yaml "gopkg.in/yaml.v2"
 )
 
-func Run() {
-	ctx := context.Background()
+type RunConfig struct {
```

Comment:
This name is confusing, a `RunConfig` implies it's a configuration for Terraform runs. Putting the name of the method into the type is redundant. There is only one type of exported config.

This should just be `Inputs`